### PR TITLE
feat: change metadata to map[string]string

### PR DESCRIPTION
**Pull request by gfyrag**
replace map[string]any by map[string]string on all metadata

### CODE REVIEW

File: components/ledger/pkg/machine/machine.go [hunk 1]
```diff
@@ -30,8 +30,8 @@ func Run(m *vm.Machine, script core.RunScript) (*Result, error) {
 
 	result := Result{
 		Postings:        make([]core.Posting, len(m.Postings)),
-		Metadata:        map[string]any{},
-		AccountMetadata: map[string]core.Metadata{},
+		Metadata:        m.GetTxMetaJSON(),
+		AccountMetadata: m.GetAccountsMetaJSON(),
```
Comment (antoinegelloz) [inline components/ledger/pkg/machine/machine.go:34]: Much simpler! 🎉

File: components/fctl/pkg/metadata.go [hunk 2]
```diff
@@ -21,19 +22,15 @@ func ParseMetadata(array []string) (map[string]any, error) {
 	return metadata, nil
 }
 
-func PrintMetadata(out io.Writer, metadata map[string]any) error {
+func PrintMetadata(out io.Writer, metadata map[string]string) error {
```
Comment (paul-nicolas) [inline components/fctl/pkg/metadata.go:25]: Why not use metadata.Metadata instead of map[string]string directly ?

Comment (gfyrag) [reply components/fctl/pkg/metadata.go:25]: You're right, i will change

File: components/ledger/pkg/storage/sqlstorage/ledger/accounts.go [hunk 3]
```diff
@@ -25,8 +26,8 @@ const (
 type Accounts struct {
 	bun.BaseModel `bun:"accounts,alias:accounts"`
 
-	Address  string         `bun:"address,type:varchar,unique,notnull"`
-	Metadata map[string]any `bun:"metadata,type:jsonb,default:'{}'"`
+	Address  string            `bun:"address,type:varchar,unique,notnull"`
+	Metadata map[string]string `bun:"metadata,type:jsonb,default:'{}'"`
```
Comment (paul-nicolas) [inline components/ledger/pkg/storage/sqlstorage/ledger/accounts.go:30]: Should we plan a SQL migration for this ? Do we know what we currently have in the database?

Comment (gfyrag) [reply components/ledger/pkg/storage/sqlstorage/ledger/accounts.go:30]: Yes we will need a proper migration for this.

**Pull request merged**